streams/src/main/java/org/apache/kafka/streams/Topology.java

```diff
@@ -783,7 +783,8 @@ public synchronized <K, V> Topology addGlobalStore(final StoreBuilder<?> storeBu
             valueDeserializer,
             topic,
             processorName,
-            () -> ProcessorAdapter.adapt(stateUpdateSupplier.get())
+            () -> ProcessorAdapter.adapt(stateUpdateSupplier.get()),
+            false
```

Comment:
These could be true as well. It would depend on if we want the reprocessing to be for only `StreamBuilder#addGlobalStore` or all `addGlobalStore`